packages/taquito/test/contract/rpc-estimate-provider.spec.ts

```diff
@@ -124,14 +125,35 @@ describe('RPCEstimateProvider test', () => {
   });
 
   describe('transfer', () => {
-    test('return the correct estimate for multiple internal origination', async done => {
+    test('return the correct estimate for multiple internal origination, reveal needed', async done => {
```

Comment:
Do we need to test for reveal op. here, whether its executed? This test only test for estimate values Line 137

Reply:
The estimation includes the fees needed for the reveal op.
This mock means that the account does not have a manager, so it needs to be revealed: mockRpcClient.getManagerKey.mockResolvedValue(null);